packages/jest-editor-support/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "jest-editor-support",
+  "version": "17.0.3",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/facebook/jest.git"
+  },
+  "license": "BSD-3-Clause",
+  "main": "build/index.js",
+  "scripts": {
+    "test": "../../packages/jest-cli/bin/jest.js"
+  },
+  "dependencies": {
+    "typescript": "^2.0.10"
```

Comment:
Originally I used Babylon as an AST parser for your test files, but after some discussion in https://github.com/orta/jest/pull/1 we figured typescript's AST covers both JS + TS cases, and so we went with it 👍